modules/hitlnext/src/config.ts

```diff
@@ -41,13 +41,18 @@ export interface Config {
 
   /**
    * @param transferMessage The message sent to the user when he is being transferred to an agent. E.g. ̀`{ "lang": "message"}`.
+   * @example { "en": "You are being transferred to an agent.", "fr": "Vous êtes transféré à un agent."}
+   * @default {}
```

Comment:
I think we should provide a default for those 2 customizable messages.

Examples are perfect to me